tests/test_data/python-sphinx_autonc.spec

```diff
@@ -13,8 +13,8 @@ Source0:        %{pypi_source}
 BuildArch:      noarch
 
 BuildRequires:  python2-devel
-BuildRequires:  (python2dist(alabaster) < 0.8 with python2dist(alabaster) >= 0.7)
-BuildRequires:  ((python2dist(babel) < 2 or python2dist(babel) > 2) with python2dist(babel) >= 1.3)
+BuildRequires:  (python2dist(alabaster) >= 0.7 with python2dist(alabaster) < 0.8)
+BuildRequires:  (python2dist(babel) >= 1.3 with (python2dist(babel) < 2 or python2dist(babel) > 2))
```

Comment:
This line looks pretty fishy though ...

Reply:
This looks like a translation of requires:babel>=1.3,!=2

Reply:
Ah, yeah that makes sense.

Reply:
OK, I squashed the last change into an earlier patch in the set.  If this looks good to you, I'll merge this branch.